deploy/undeploy_all.sh

```diff
@@ -20,3 +22,14 @@ $KUBE_CLI delete -f $DEPLOY_PATH/election_role.yaml
 $KUBE_CLI delete -f $DEPLOY_PATH/election_role_binding.yaml
 $KUBE_CLI delete -f $DEPLOY_PATH/operator_config.yaml
 $KUBE_CLI delete -f $DEPLOY_PATH/operator.yaml
+
+$KUBE_CLI delete -f $DEPLOY_PATH/Issuer_activemq-artemis-selfsigned-issuer.yaml
+$KUBE_CLI delete -f $DEPLOY_PATH/Certificate_activemq-artemis-serving-cert.yaml
+$KUBE_CLI delete -f $DEPLOY_PATH/MutatingWebhookConfiguration_activemq-artemis-mutating-webhook-configuration.yaml
+$KUBE_CLI delete -f $DEPLOY_PATH/ValidatingWebhookConfiguration_activemq-artemis-validating-webhook-configuration.yaml
+$KUBE_CLI delete -f $DEPLOY_PATH/Service_activemq-artemis-webhook-service.yaml
+$KUBE_CLI delete secret webhook-server-cert
+
+rm tls.crt
```

Comment:
I think the generated files should be deleted right after creating the secret.

Reply:
agreed. will update.